moler/cmd/at/set_cell_id_nr.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+AT+C5GREG
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Jakub Kochaniak'
+__copyright__ = 'Copyright (C) 2023, Nokia'
+__email__ = 'jakub.kochaniak@nokia.com'
+
+from moler.cmd.at.genericat import GenericAtCommand
+from moler.exceptions import WrongUsage
+
+
+class SetCellIdNr(GenericAtCommand):
+    """
+    Command to set NR cell registration.
+
+    Possible options for selected_mode, but verify with latest 3gpp specification 27.007:
```

Comment:
Please move to `__init__` as docstring to `selected_mode` parameter

Reply:
Yes, I agree. Calling an object constructed has quick-info rendered from `__init__` rather than class docstring.
Thanks for this! It is done as [here](https://github.com/nokia/moler/pull/472#discussion_r1265714513)